lib/rules/no-dupe-disjunctions.ts

```diff
@@ -108,7 +108,7 @@ function containsAssertions(expression: NoParent<Expression>): boolean {
             },
         })
         return false
-    } catch (_error) {
+    } catch {
```

Comment:
As of https://github.com/typescript-eslint/typescript-eslint/pull/8971, these are now flagged by default by `@typescript-eslint/no-unused-vars`. They were unused before.